run_server.sh

```diff
@@ -70,6 +71,10 @@ if [[ $FLASK_ENV != "local" ]]; then
 fi
 echo "Starting localhost with FLASK_ENV='$FLASK_ENV' on port='$PORT'"
 
+# Copy over custom DC files
+find ./server/templates/custom_dc/custom ! -name '.empty' -type f -exec rm -f {} +
+cp -rf custom_dc/*.html server/templates/custom_dc/custom/
```

Comment:
we should add a note in documentation that hot reloads won't work and they would need to rerun this script on each change.